packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -1569,6 +1633,13 @@ export class MergeTree {
                 newSegment.seq = seq;
                 newSegment.localSeq = localSeq;
                 newSegment.clientId = clientId;
+                if (this.options?.attribution?.track && seq !== UnassignedSequenceNumber) {
```

Comment:
should the system support getting the current user info for local changes? maybe add it later, but seem like a pain for application to support

Reply:
yeah, I was planning on making this more ergonomic later. if that were done here, we'd need some kind of sentinel value or something that attributor implementations always returned the current user on (since all the information stored here merely associates content to sequenceNumbers, and current user hasn't necessarily had any of their edits persisted into the document yet)

Reply:
"attribution" API on ISegment is marked @alpha now, so we can change the semantics around it if we want to break it. alternatively adding another layer which handles this is always an option.